src/viewer/controls.styl

```diff
@@ -0,0 +1,121 @@
+@require 'components/selectionbar.styl'
+@require 'settings/breakpoints.styl'
+@require 'settings/palette.styl'
+
+.pho-viewer-controls
+    display         flex
+    flex-direction  row
+    justify-content center
+    align-items center
+    width           100%
+    height          100%
```

Comment:
u-flex u-flex-row u-flex-justify-center ... ? :D

Reply:
I'd rather not convert everything to utility classes in this PR. I did it for the css I touched, but there's lots more that could be improved.